src/inmanta/server/services/resourceservice.py

```diff
@@ -592,6 +592,12 @@ async def resource_action_update(
         *,
         connection: Optional[Connection] = None,
     ) -> Apireturn:
+        """
+        :param is_increment_notification: is this the increment calucation setting the deployed status,
+            instead of an actual deploy? Used to keep track of the last_success field on the resources,
+            which should not be updated for increments.
```

Comment:
Shouldn't this be documented on `methods.resource_action_update` instead?

Reply:
no, this argument doens't exist on the rest api, only internally.